File: gpao-tex-ui/source/main/src/app/services/production/models/production-heure-value.ts
/* tslint:disable */
/* eslint-disable */
import { ProductionHeureDetailsValue } from '../models/production-heure-details-value';
export interface ProductionHeureValue {
  chaineId?: number;
  date?: string;
  elementPlanningId?: number;
  id?: number;
  observation?: string;
  ofId?: number;
  partieInterreseId?: number;
  periode?: string;
  productionHeureDetails?: Array<ProductionHeureDetailsValue>;
  produitId?: number;
  quantite?: number;
}

File: gpao-tex-core/production/src/main/java/com/gpro/consulting/production/dto/ProductionHeureValue.java
package com.gpro.consulting.production.dto;

import java.util.Calendar;
import java.util.List;

public record ProductionHeureValue(
        Long id,
        Calendar date,
        Long quantite,
        String observation,
        Long elementPlanningId,
        Long ofId,
        Long chaineId,
        Long partieInterreseId,
        Long produitId,
        String periode,
        List<ProductionHeureDetailsValue> productionHeureDetails
) {
}
